### PR TITLE
Updates surgical tool descriptions

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Title.

## Why It's Good For The Game
1. LORE

2. name = "retractor"
    desc = "Retracts stuff."

I rest my case.

## Testing
- loaded in
- observed surgical tools
- descriptions updated

## Changelog
:cl:
tweak: Updated surgical tool descriptions
/:cl:

### CODE REVIEW

**Comment by SynthTwo** [comment]
Finally descriptions that make sense

**Review by SynthTwo — APPROVED** [review]
Looks good to me. Grammar is fine. What more could you want?

File: code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -271,7 +274,7 @@
 
 /obj/item/surgical_drapes
 	name = "surgical drapes"
-	desc = "Nanotrasen brand surgical drapes provide optimal safety and infection control."
+	desc = "Apearature brand surgical drapes providing privacy and infection control."
```
Comment (alex-gh) [inline code/modules/surgery/tools.dm:277]: Typo or some kind of reference I'm missing? Did you mean "Aperture"?

Comment (PopeDaveThe3th) [reply code/modules/surgery/tools.dm:277]: It's legally distinct Aperature, since they were a shower curtain company in Portal lore.

**Review by Henri215 — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Pull request merged**